packages/jest-expo/src/preset/expoModules.js

```diff
@@ -227,19 +227,6 @@ module.exports = {
             key: 'getMobileNetworkCodeAsync',
           },
         ],
-        ExpoClipboard: [
```

Comment:
@tsapeta Can we remove it from here? Is it not used for anything else?

Reply:
Even if you remove it here, it would be overridden as part of the release process and generating the mocks using `jest-expo-mock-generator` app. So, it doesn't make much sense to do now.